### PR TITLE
CAMEL-8419 Camel StreamCache does not work with CXF consumer for InOut messages

### PR DESCRIPTION
**Pull request by snurmine**
* Closing UnitOfWork after Continuation has been processed

### CODE REVIEW

File: components/camel-cxf/src/main/java/org/apache/camel/component/cxf/CxfConsumer.java [hunk 1]
```diff
@@ -47,6 +48,7 @@
 import org.slf4j.LoggerFactory;
 
 
+
```
Comment (onders86) [inline components/camel-cxf/src/main/java/org/apache/camel/component/cxf/CxfConsumer.java]: empty line

**Comment by onders86** [comment]
LGTM, as a side note, i only suspect only the corner case of when the contiunation is suspeded for some time and if somehow the continuation is resumed and timed-out . I am not sure in such async scenarios, like continuation is suspended and resumed again, stream cache will be handled and UoW will be done. This would require hard unit testing. 

And my second note,  CxfRsInvoker may require such a change too. Could you also check for that?

**Comment by snurmine** [comment]
Thanks for the feedback! It seems that synchronous case doesn't work either. I'll try to fix it.

**Comment by snurmine** [comment]
Added more tests for sync and async cases.

**Pull request closed without merging**